x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_table.ts

```diff
@@ -72,10 +73,13 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       const cols = await PageObjects.datasetQuality.parseDatasetTable();
       const datasetNameCol = cols[PageObjects.datasetQuality.texts.datasetNameColumn];
       await datasetNameCol.sort('descending');
-      const datasetNameColCellTexts = await datasetNameCol.getCellTexts();
-      expect(datasetNameColCellTexts).to.eql(
-        [apacheAccessDatasetHumanName, ...datasetNames].reverse()
-      );
+
+      await retry.tryForTime(10000, async () => {
+        const datasetNameColCellTexts = await datasetNameCol.getCellTexts();
+        expect(datasetNameColCellTexts).to.eql(
+          [apacheAccessDatasetHumanName, ...datasetNames].reverse()
+        );
+      });
```

Comment:
The error

```
fail: serverless observability UI Dataset Quality Dataset quality table shows sort by dataset name and show namespace
--
  | │       Error: expected [ 'synth.3', 'synth.2', 'synth.1', 'apache.access' ] to sort of equal [ 'synth.3',
  | │   'synth.2',
  | │   'synth.1',
  | │   'Apache access logs' ]
  | │       + expected - actual
  | │
  | │        [
  | │          "synth.3"
  | │          "synth.2"
  | │          "synth.1"
  | │       -  "apache.access"
  | │       +  "Apache access logs"
  | │        ]
```
likely happens when the integration hasn't been loaded successfully. Will this retry address it given it won't reload the data?